usmqe_tests/rpm/packagelist.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf8 -*-
+
+tendrl_packages = [
+    "tendrl-alerting",
+    "tendrl-api",
+    "tendrl-api-httpd",
+    "tendrl-ceph-integration",
+    "tendrl-commons",
+    "tendrl-dashboard",
+    "tendrl-gluster-integration",
+    "tendrl-node-agent",
+    "tendrl-node-monitoring",
+    "tendrl-performance-monitoring",
+    ]
```

Comment:
@mbukatov what is the reason for hardcoded list of `tendrl_packages`? Would it be possible to use similar dynamic approach as for Tendrl dependencies?